hack/runtime-migrator/cmd/restore/restore.go

```diff
@@ -103,26 +120,134 @@ func (r Restore) Do(ctx context.Context, runtimeIDs []string) error {
 		}
 
 		slog.Info("Runtime restore performed successfully", "runtimeID", runtimeID)
-		r.results.OperationSucceeded(runtimeID, currentShoot.Name)
+		r.results.OperationSucceeded(runtimeID, currentShoot.Name, appliedCRBs, appliedOIDC)
 	}
 
 	resultsFile, err := r.outputWriter.SaveRestoreResults(r.results)
 	if err != nil {
 		return err
 	}
 
-	slog.Info(fmt.Sprintf("Restore completed. Successfully restored backups: %d, Failed operations: %d", r.results.Succeeded, r.results.Failed))
+	slog.Info(fmt.Sprintf("Restore completed. Successfully restored backups: %d, Failed operations: %d, Skipped backups: %d, ", r.results.Succeeded, r.results.Failed, r.results.Skipped))
```

Comment:
Please also align report for skipped backups with backup.go file. Currently in dry-run mode skipped backups are not reported correctly.

`2025/01/07 18:08:18 INFO Backup completed. Successfully stored backups: 0, Failed backups: 0`